### PR TITLE
interpretert isn't a messaget

### PR DESCRIPTION
**Pull request by tautschnig**
Use a messaget member instead as there is no is-a relationship between
interpretert and messaget.

- [x] Each commit message has a non-empty body, explaining why the change was made.
- n/a Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- n/a The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- n/a My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- n/a White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

File: src/goto-programs/interpreter_class.h [hunk 1]
```diff
@@ -24,14 +24,14 @@ Author: Daniel Kroening, kroening@kroening.com
...
-class interpretert:public messaget
+class interpretert
 {
 public:
   interpretert(
     const symbol_tablet &_symbol_table,
     const goto_functionst &_goto_functions,
     message_handlert &_message_handler)
-    : messaget(_message_handler),
+    : output(_message_handler),
```
Comment (hannes-steffenhagen-diffblue) [inline src/goto-programs/interpreter_class.h:34]: Why is this called `output` rather than `log` here?

**Review by hannes-steffenhagen-diffblue — APPROVED** [review]
Naming seems inconsistent, but otherwise OK

**Comment by hannes-steffenhagen-diffblue** [comment]
GH Actions failed due to network error, restarting (all of them because apparently there's no way to restart only a single one).

File: src/goto-programs/interpreter_class.h (hunk 1 above)
Comment (tautschnig) [reply src/goto-programs/interpreter_class.h:34]: The interpreter necessarily provides user-facing output, i.e., is not useful if not printing some of the messages. So I thought "output" may be a better choice than "log" on this occasion. But I might well be wrong?

**Review by peterschrammel — APPROVED** [review]
Fine with output or log.

**Pull request merged**